src/main/kotlin/demo/loader/FakeRecruitmentProcess.kt

```diff
@@ -41,7 +41,7 @@ object FakeRecruitmentProcess {
     fun getProcesses(offers: List<Offer>): List<RecruitmentProcess> {
         return offers.map {
             ProcessPayload(
-                recruitmentStages = recruitmentStages.subList(0, 1+getRandomNumber(1).toInt())
+                recruitmentStages = recruitmentStages().subList(0, 3)
```

Comment:
Magic numbers - co to jest to 3? (Tu akurat serio pytam XD)

Reply:
Bo stwierdziłem początkowo można wziąć 3 stage,  ale mogę wrócić do randomwa bo wyszło że problem był z innego powodu.